### PR TITLE
fixed v2 invisible support

### PR DESCRIPTION
**Pull request by therealwolf42**
Similar to #25, V2 invisible Recaptcha is now fully supported.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,15 +49,29 @@ using top level options
 {
   // ...
   recaptcha: {
-    hideBadge: Boolean, // Hide badge element (v3)
+    hideBadge: Boolean, // Hide badge element (v3 & v2 via size=invisible)
     language: String,   // Recaptcha language (v2)
     siteKey: String,    // Site key for requests
     version: Number     // Version
```
Comment (mvrlin) [inline README.md]: You forgot the comma 😁

Comment (mvrlin) [inline README.md]: You forgot the comma 😁

Comment (therealwolf42) [reply README.md]: Fixed :)

**Comment by Clayboy** [comment]
+1

**Comment by Clayboy** [comment]
can someone merge this PR in order to have a full feature recaptcha v2 ?

**Comment by martinshumberto** [comment]
+1

**Comment by mvrlin** [comment]
Thank you for your PR. Sorry for late response.

**Pull request merged**